src/modal/mode/desktop/PeraWalletConnectModalDesktopMode.ts

```diff
@@ -223,6 +192,75 @@ export class PeraWalletModalDesktopMode extends HTMLElement {
     }
   }
 
+  handleChangeView() {
+    const downloadPeraButton = this.shadowRoot?.getElementById(
+      "pera-wallet-connect-modal-desktop-mode-download-pera-button"
+    );
+    const backButton = this.shadowRoot?.getElementById(
+      "pera-wallet-connect-modal-download-pera-view-back-button"
+    );
+    const webWalletLaunchButton = this.shadowRoot?.getElementById(
+      "pera-wallet-connect-web-wallet-launch-button"
+    );
+
+    if (downloadPeraButton) {
+      downloadPeraButton.addEventListener("click", () => {
+        this.onClickDownload();
+      });
+    }
+
+    if (backButton) {
+      backButton.addEventListener("click", () => {
+        this.onClickBack();
+      });
+    }
+
+    if (webWalletLaunchButton) {
+      webWalletLaunchButton.addEventListener("click", () => {
+        this.webWalletConnect();
+      });
+    }
+  }
+
+  initDownloadLinks() {
+    const downloadIOS = this.shadowRoot?.getElementById(
+      "pera-wallet-connect-modal-download-pera-view-download-ios-link"
+    );
+    const downloadAndroid = this.shadowRoot?.getElementById(
+      "pera-wallet-connect-modal-download-pera-view-download-android-link"
+    );
+    const downloadAPK = this.shadowRoot?.getElementById(
+      "pera-wallet-connect-modal-download-pera-view-download-apk-link"
+    );
+
+    if (downloadIOS) {
+      downloadIOS.addEventListener("click", () => {
```

Comment:
Can't we use anchor element for this purpose?

Reply:
I was using anchor element but it wasn't working in a way I don't know why 🤔